### PR TITLE
Fix error prone issues pointed out by Codacy in JS files

### PR DESCRIPTION
**Pull request by beatrycze-volk**
Small quality improvements

### CODE REVIEW

File: Resources/Public/Javascript/PageView/Utility.js [hunk 1]
```diff
@@ -760,7 +760,7 @@ dlfUtils.isCorsEnabled = function (imageObjs) {
  * @return {boolean}
  */
 dlfUtils.isWebGLEnabled = function () {
-    if (!!window.WebGLRenderingContext) {
```
Comment (albig) [inline Resources/Public/Javascript/PageView/Utility.js]: Actually this changes logic of this function. Please don't ask me, why this is double-negated. Probably the explanation is found here: https://stackoverflow.com/questions/10467475/double-negation-in-javascript-what-is-the-purpose

This isWebGLEnabled function is used with the imagemanipulation tool. If WebGL is not avaiable, the tool is hidden.

**Review by albig — CHANGES_REQUESTED** [review]

File: Resources/Public/Javascript/PageView/Utility.js (hunk 1 above)
Comment (beatrycze-volk) [reply Resources/Public/Javascript/PageView/Utility.js]: Answer in this question is not related to this exact case, as there double negation is not inside the `if` condition. Codacy suggest that inside `if` block it is not necessary to make double negation: _In contexts such as an if statement's test where the result of the expression will already be coerced to a Boolean, casting to a Boolean via double negation (!!) or a Boolean call is unnecessary._

Should I anyway reverse this change?

Comment (albig) [reply Resources/Public/Javascript/PageView/Utility.js]: Yes, please revert. The suggestion from Codacy seems to be not true in this case. Have a look at this older question:

https://stackoverflow.com/questions/11871077/proper-way-to-detect-webgl-support

And have a try with this jsfiddle:

https://jsfiddle.net/albig/r58yuxmt/7/

If we keep your change, the function does not work as expected.

Comment (beatrycze-volk) [reply Resources/Public/Javascript/PageView/Utility.js]: Reversed

**Review by albig — APPROVED** [review]

**Pull request merged**